apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1006,6 +1017,116 @@ def _chunk_is_contained_in_axis(
     return True
 
 
+def _maybe_ingest_uns(
+    m: Measurement,
+    uns: Mapping[str, object],
+    platform_config: Optional[PlatformConfig],
+    ingest_mode: IngestMode,
+    *,
+    use_relative_uri: Optional[bool],
+) -> None:
+    # Don't try to ingest an empty uns.
+    if not uns:
+        return
+    _ingest_uns_dict(
+        m, "uns", uns, platform_config, ingest_mode, use_relative_uri=use_relative_uri
+    )
+
+
+def _ingest_uns_dict(
+    parent: AnyTileDBCollection,
+    parent_key: str,
+    dct: Mapping[str, object],
+    platform_config: Optional[PlatformConfig],
+    ingest_mode: IngestMode,
+    *,
+    use_relative_uri: Optional[bool],
+) -> None:
+    with _create_or_open_coll(
+        Collection, _util.uri_joinpath(parent.uri, parent_key), ingest_mode
+    ) as coll:
+        _maybe_set(parent, parent_key, coll, use_relative_uri=use_relative_uri)
+        coll.metadata["soma_tiledbsoma:type"] = "uns"
+        for key, value in dct.items():
+            if key == "rank_genes_groups":
+                # This is a structured array, which we do not support.
+                logging.log_io(None, f"skipping uns structured array {key!r}")
```

Comment:
this is worth a message at the lower-verbosity level -- only on exceptional occasions will people be using the debug level

Reply:
@thetorpedodog please update & i'll accept :)

Reply:
unless this is stale but this PR has only a single commit so i'm not sure how it could have changed since i previously reviewed, unless you force-pushed which i also don't see 🤔